src/patternfly/components/Pagination/pagination.scss

```
@@ -1,6 +1,11 @@
 $pf-c-pagination--breakpoint-map: build-breakpoint-map();
+$pf-c-pagination--variable-map: build-spacer-map("none", "md", "xl");
```

This should probably support all of the spacers

```suggestion
$pf-c-pagination--variable-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl");
```